eventmesh-connectors/eventmesh-connector-http/src/main/java/org/apache/eventmesh/connector/http/sink/connector/HttpSinkConnector.java

```diff
@@ -0,0 +1,134 @@
+import lombok.extern.slf4j.Slf4j;
+
+
+@Slf4j
+public class HttpSinkConnector implements Sink {
+
+    private HttpSinkConfig httpSinkConfig;
+
+    private WebClient webClient;
+
+    private volatile boolean isRunning = false;
+
+    @Override
+    public Class<? extends Config> configClass() {
+        return HttpSinkConfig.class;
+    }
+
+    @Override
+    public void init(Config config) throws Exception {
+        httpSinkConfig = (HttpSinkConfig) config;
+        doInit();
+    }
+
+    @Override
+    public void init(ConnectorContext connectorContext) throws Exception {
+        SinkConnectorContext sinkConnectorContext = (SinkConnectorContext) connectorContext;
+        this.httpSinkConfig = (HttpSinkConfig) sinkConnectorContext.getSinkConfig();
+        doInit();
+    }
+
+    @SneakyThrows
+    private void doInit() {
+        final Vertx vertx = Vertx.vertx();
+        // TODO Add more configurations
+        WebClientOptions options = new WebClientOptions()
+            .setDefaultHost(this.httpSinkConfig.connectorConfig.getHost())
+            .setDefaultPort(this.httpSinkConfig.connectorConfig.getPort())
+            .setSsl(this.httpSinkConfig.connectorConfig.isSsl())
+            .setIdleTimeout(this.httpSinkConfig.connectorConfig.getIdleTimeout());
+        this.webClient = WebClient.create(vertx, options);
+    }
+
+    @Override
+    public void start() throws Exception {
+        this.isRunning = true;
+    }
+
+    @Override
+    public void commit(ConnectRecord record) {
+
+    }
+
+    @Override
+    public String name() {
+        return this.httpSinkConfig.connectorConfig.getConnectorName();
+    }
+
+    @Override
+    public void stop() throws Exception {
+        this.isRunning = false;
+        this.webClient.close();
+    }
+
+    public boolean isRunning() {
+        return isRunning;
+    }
+
+    @Override
+    public void put(List<ConnectRecord> sinkRecords) {
+        for (ConnectRecord sinkRecord : sinkRecords) {
+            try {
+                if (Objects.isNull(sinkRecord)) {
+                    log.warn("ConnectRecord data is null, ignore.");
+                    continue;
+                }
+                sendMessage(sinkRecord);
+            } catch (Exception e) {
+                log.error("Failed to sink message via HTTP. ", e);
+            }
+        }
+    }
+
+    private void sendMessage(ConnectRecord record) {
+        this.webClient.post(this.httpSinkConfig.connectorConfig.getPath())
+            .putHeader("Content-Type", "application/json; charset=utf-8")
+            .sendJson(record, ar -> {
+                if (ar.succeeded()) {
+                    if (ar.result().statusCode() != HttpResponseStatus.OK.code()) {
+                        log.error("[HttpSinkConnector] Failed to send message via HTTP. Response: {}", ar.result());
+                    } else {
+                        log.info("[HttpSinkConnector] Successfully send message via HTTP. ");
+                    }
```

Comment:
Actually, since the sink connector has received a response, the message sending step can be considered success, but the message delivary failed. Better print the message ID in the log when success and response body when failure.

Reply:
I didn't find any ID-related fields in the ConnectRecord class, so I'm planning to directly log the entire ConnectRecord object

Reply:
We can consider offset and timestamp as unique identifiers for a message.